### PR TITLE
This commit will complete task 8 and the respective tests

### PR DESCRIPTION
**Pull request by AndreiThira**

### CODE REVIEW

File: models/topic-model.js [hunk 1]
```diff
@@ -72,9 +72,22 @@ const fetchPostCommentByArticle = (username, body, article_ID, next) => {
...
+    return checkExists("articles", "article_id", article_ID)
+    .then(() => {
+      const query = "UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *;"
+      const values = [newVotes, article_ID]
+      return db.query(query, values).then((comment)=>{
+        return comment.rows
+      })
+      .catch(next)
+})
+.catch(next)
```
Comment (nc-poppy-m) [inline models/topic-model.js]: We shouldn't need these catch blocks in the model

File: endpoints.json [hunk 2]
```diff
@@ -80,5 +80,4 @@
       ]
     }
   }
-
```
Comment (nc-poppy-m) [inline endpoints.json:83]: Remember to add in your new endpoint :)

**Review by nc-poppy-m — CHANGES_REQUESTED** [review]
Great work! Just see my comments in the model. If you like, you could also consider what happens if we send an object which doesn't have an inc_votes key, but this isn't mandatory. Just to make sure it works, remove the catch blocks in the model and resubmit the pr with all tests passing 🙂

**Review by mezzDavies — APPROVED** [review]
Grand - merge away!

**Pull request merged**